test/src/e2e_vm_tests/test_programs/should_fail/vec_set_index_out_of_bounds/Cargo.toml

```diff
@@ -0,0 +1,15 @@
+[project]
```

Comment:
This file is not needed when adding tests under `e2e_vm_tests` because we rely on `forc` directly for these tests (and not the SDK)